packages/schema/bind/package.json

```diff
@@ -4,10 +4,11 @@
   "license": "MIT",
   "main": "build/index.js",
   "scripts": {
-    "build": "rimraf ./build && tsc --project tsconfig.build.json",
+    "build": "rimraf ./build && tsc --project tsconfig.build.json && yarn copy:templates",
     "lint": "eslint --color --ext .ts src/",
     "test": "jest --passWithNoTests --runInBand --verbose",
-    "test:watch": "jest --watch --passWithNoTests --verbose"
+    "test:watch": "jest --watch --passWithNoTests --verbose",
+    "copy:templates": "copyfiles -u 1 src/**/*.mustache build/"
```

Comment:
so annoying that tsc doesn't have copy files option :(

Reply:
I wonder if it would be better to keep them out of src

Reply:
Agreed, would be super helpful to just have this as a tsconfig option.

I'd prefer to keep the templates along-side the source, to help with readability of the code-base. I'm also considering making all `.mustache` files JS modules that export a string. This way file system access is required at run-time.